blogs/vonage-solaria-python-fastapi/src/server.py

```
@@ -0,0 +1,449 @@
+                    "Content-Type": "application/json",
+                },
+                timeout=10,
+            )
+        
+        # Log the response for debugging
+        logger.debug(f"Gladia API response status: {r.status_code}")
+        
+        try:
+            resp_json = r.json()
+            logger.debug(f"Gladia API response: {json.dumps(resp_json)}")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)